src/slack_interaction_api_util.ts

```diff
@@ -17,14 +17,12 @@ export async function replaceSlackMessageBlocks({
   logger.info('ENTERING SLACKINTERACTIONAPIUTIL.replaceSlackMessageBlocks');
   // Replace voter status panel with message.
   const response = await SlackApiUtil.slackAPI.post('chat.update', {
-    params: {
-      channel: slackChannelId,
-      token: process.env.SLACK_BOT_ACCESS_TOKEN,
-      ts: slackParentMessageTs,
-      blocks: newBlocks,
-    },
+    'Content-type': 'application/json',
```

Comment:
application/json is built into slackAPI here: https://github.com/tomerovadia/VoterHelpLine/blob/master/src/slack_api_util.ts#L17

so you can omit that